### PR TITLE
GEODE-9870: Avoid Radish MOVED error during flushAll in AuthWhileServersRestartDUnitTest

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
- Since servers have been stopped and restarted, as part of the test,
  buckets may be moving. Occasionally this may still be the case when
  the test is done and flushAll is cleaning up. To avoid that, make sure
  that everything is rebalanced before flushAll is called.

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### CODE REVIEW

**Review by dschneider-pivotal — APPROVED** [review]

File: geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/auth/AuthWhileServersRestartDUnitTest.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
   @ClassRule
   public static ExecutorServiceRule executor = new ExecutorServiceRule();
 
+  private static MemberVM server1;
```
Comment (ringles) [inline geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/auth/AuthWhileServersRestartDUnitTest.java:53]: Sorry, where is this variable used? I see it being set, but I don't see where the value gets used.

**Review by DonalEvans — APPROVED** [review]

**Review by ringles — CHANGES_REQUESTED** [review]
Intellij can't find where the "server1" variable is used, either. Can it be removed?

File: geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/auth/AuthWhileServersRestartDUnitTest.java (hunk 1 above)
Comment (jdeppe-pivotal) [reply geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/auth/AuthWhileServersRestartDUnitTest.java]: Thanks. I was using it but that changed... It's gone now.

**Review by ringles — APPROVED** [review]
Looks good!

**Pull request merged**